src/services/TradeService.sol

```diff
@@ -374,7 +374,12 @@ contract TradeService is ReentrancyGuardUpgradeable, ITradeService, OwnableUpgra
       // in order to avoid bypassing the maximum profit cap.
       // Additionally, if the minimum profit duration is active, increasing the position is not allowed.
       // This is checked by comparing _delta to 0, as it is virtually impossible for _delta to be 0 if the position is active without a minimum profit duration.
-      if (_isProfit && (_delta >= _vars.position.reserveValueE30 || _delta == 0)) {
+      uint256 minProfitDuration = _vars.configStorage.minProfitDurations(_marketIndex);
+      if (
+        _isProfit &&
+        (_delta >= _vars.position.reserveValueE30 ||
+          (block.timestamp < _vars.position.lastIncreaseTimestamp + minProfitDuration))
+      ) {
         revert ITradeService_NotAllowIncrease();
       }
```

Comment:
Adding discussion from the call today:

Currently the `minProfitDuration` only applies to profitable positions, so if someone can manipulate prices in the short-term, maybe they can turn a big loss into a tiny loss and cash out (and perhaps can be used as a "free option" strategy).

So, it could be worth considering disallowing increases and decreases *regardless* of the position's profit:

```solidity
uint256 minProfitDuration = _vars.configStorage.minProfitDurations(_marketIndex);
if (
  (_isProfit && _delta >= _vars.position.reserveValueE30) ||
    (block.timestamp < _vars.position.lastIncreaseTimestamp + minProfitDuration)
) {
  revert ITradeService_NotAllowIncrease();
}
```

and for `_decreasePosition()`:

```diff
  uint256 minProfitDuration = ConfigStorage(configStorage).minProfitDurations(_marketIndex);
- if (isProfit && block.timestamp < (_vars.position.lastIncreaseTimestamp + minProfitDuration)) {
+ if (block.timestamp < (_vars.position.lastIncreaseTimestamp + minProfitDuration)) {
    revert ITradeService_NotAllowDecrease();
  }
```